lib/monetize_api.ts

```diff
@@ -36,8 +36,8 @@ export namespace MonetizeEnums {
     export enum Networks {
         IronSource = 'ironSource',
         IronSourceBidding = 'ironSourceBidding',
-        Vungle = 'Vungle',
-        VungleBidding = 'vungleBidding',
+        LiftoffMonetize = 'liftoffMonetize',
```

Comment:
The 'Vungle' enum value was mistakenly changed to 'liftoffMonetize'. Please revert it back to 'Vungle'.